src/Neo.VM/Types/Buffer.cs

```diff
@@ -112,5 +113,14 @@ public override string ToString()
         {
             return GetSpan().TryGetString(out var str) ? $"(\"{str}\")" : $"(\"Base64: {Convert.ToBase64String(GetSpan())}\")";
         }
+
+        /// <summary>
+        /// Invalidate
+        /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void Invalidate()
```

Comment:
Is this a good practice?
Then we will need a specific check for checking is it is 0.
Why not null for example?

Reply:
@shargon added, but it `int` cant be `null`, plus its cached

Reply:
Invalidate reset the hash method, is only in buffer items

Reply:
Here a simple one
```csharp
public override int GetHashCode()
{
    if (byteArray == null || byteArray.Length == 0)
        return 0;

    unchecked // Overflow is fine, just wrap
    {
        int hash = 17; // Seed with a prime number
        foreach (byte b in byteArray)
        {
            hash = hash * 31 + b; // Multiply by a prime number and add byte value
        }
        return hash;
    }
}
```